core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -142,12 +142,13 @@ private TableProperties() {}
 
   public static final String PARQUET_COMPRESSION = "write.parquet.compression-codec";
   public static final String DELETE_PARQUET_COMPRESSION = "write.delete.parquet.compression-codec";
-  public static final String PARQUET_COMPRESSION_DEFAULT = "gzip";
+  public static final String PARQUET_COMPRESSION_DEFAULT = "zstd";
 
   public static final String PARQUET_COMPRESSION_LEVEL = "write.parquet.compression-level";
   public static final String DELETE_PARQUET_COMPRESSION_LEVEL =
       "write.delete.parquet.compression-level";
-  public static final String PARQUET_COMPRESSION_LEVEL_DEFAULT = null;
+  public static final String PARQUET_COMPRESSION_LEVEL_DEFAULT =
+      null; // For zstd, it is default to "3"
```

Comment:
How valuable is this comment given than it is specific to the underlying codec and may change in the future?

Reply:
Agree, like we don't mention gzip is default to "6".

Reply:
Removed.